runelite-client/src/main/java/net/runelite/client/plugins/deathindicator/DeathIndicatorConfig.java

```diff
@@ -0,0 +1,180 @@
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.deathindicator;
+
+import java.time.Instant;
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+	keyName = "deathIndicator",
+	name = "Death Indicator",
+	description = "Configuration for the death indicator plugin"
+)
+public interface DeathIndicatorConfig extends Config
+{
+	@ConfigItem(
+		position = 1,
+		keyName = "deathHintArrow",
+		name = "Death Hint Arrow",
+		description = "Configures whether or not to show a hint arrow to death location"
+	)
+	default boolean showDeathHintArrow()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 2,
+		keyName = "deathInfoBox",
+		name = "Death InfoBox",
+		description = "Configures whether or not to show item reclaim timer and death world infobox"
+	)
+	default boolean showDeathInfoBox()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 3,
+		keyName = "deathOnWorldMap",
+		name = "Mark on World Map",
+		description = "Configures whether or not to show death location on the world map"
+	)
+	default boolean showDeathOnWorldMap()
+	{
+		return true;
+	}
+
+	// Stored Data
+	@ConfigItem(
+		keyName = "hasDied",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default boolean hasDied()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "hasDied",
+		name = "",
+		description = ""
+	)
+	void hasDied(boolean hasDied);
+
+	@ConfigItem(
+		keyName = "deathWorld",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default int deathWorld()
+	{
+		return -1;
+	}
+
+	@ConfigItem(
+		keyName = "deathWorld",
+		name = "",
+		description = ""
+	)
+	void deathWorld(int deathWorld);
+
+	@ConfigItem(
+		keyName = "deathLocationX",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default int deathLocationX()
```

Comment:
Change deathLocationX and deathLocationY to Point deathLocation

Reply:
Fixed. Changed deathLocationX and deathLocationY to Point deathLocationPoint.